simapp/app_di.go

```diff
@@ -312,6 +326,23 @@ func (app *SimApp) setCustomAnteHandler() {
 	app.SetAnteHandler(anteHandler)
 }
 
+func (app *SimApp) Precommiter(ctx sdk.Context) {
+	if err := app.ModuleManager.Precommit(ctx); err != nil {
+		panic(err)
+	}
+
+	app.UnorderedTxManager.OnNewBlock(ctx.BlockTime())
+}
+
+// Close implements the Application interface and closes all necessary application
```

Comment:
Ditto, should be deleted, runtime abstracts it

Reply:
ah, didn't realize it was there already